DashAI/back/metrics/classification_metric.py

```diff
@@ -21,3 +25,25 @@ def validate_inputs(true_labels: list, pred_labels: list):
     """
     if len(true_labels) != len(pred_labels):
         raise ValueError("The length of the true and predicted labels must be equal.")
+
+
+def prepare_to_metric(true_labels: DashAIDataset, probs_pred_labels: list):
```

Comment:
`probs_pred_labels` should not be an np.array?
If so, you should change the description to something like "A two-dimensional matrix in which each column represents a class and the row values represent the probability that an example belongs to the class associated with the column

Also, I think that is good idea to change `true_labels` to `dataset`, since that is what you are finally providing to the method. If so, you should also update the docstring definition (something like "A dashai dataset with labels included".


```suggestion
def prepare_to_metric(dataset: DashAIDataset, probs_pred_labels: np.ndarray) -> Tuple[np.array, np.array]:
```